packages/mask/src/plugins/Trader/apis/1inch/index.ts

```diff
@@ -0,0 +1,34 @@
+import BigNumber from 'bignumber.js'
+import { first } from 'lodash-unified'
+import { BIPS_BASE, ONE_INCH_BASE_URL } from '../../constants'
+import type {
+    SwapOneErrorResponse,
+    SwapQuoteOneResponse,
+    SwapQuoteOneRequest,
+    SwapOneServerErrorResponse,
+    SwapOneValidationErrorResponse,
+} from '../../types'
+import type { NetworkType } from '@masknet/web3-shared-evm'
+import urlcat from 'urlcat'
+
+export async function swapOneQuote(request: SwapQuoteOneRequest, networkType: NetworkType) {
+    const params: Record<string, string | number> = {}
+    Object.entries(request).map(([key, value]) => {
+        params[key] = value
+    })
+    if (request.slippage) params.slippage = new BigNumber(request.slippage).dividedBy(BIPS_BASE).toFixed()
+
+    const response = await fetch(urlcat(ONE_INCH_BASE_URL[networkType], 'quote' || 'swap', params))
```

Comment:
1. `'quote' || 'swap'` always returns `'quote'`, is it a mistake?
2. We have a `fetchJSON<T>`, try using it.